bundled/tool/zenml_wrappers.py

```
@@ -595,7 +603,25 @@
         """
         try:
             run_id = args[0]
-            run = self.client.get_pipeline_run(run_id)
+            # Get pipeline run with hydration to ensure we have step data
+            run = self.client.get_pipeline_run(run_id, hydrate=True)
+            
+            # Check if steps are available for graph generation
+            if not hasattr(run, 'steps') or not run.steps:
+                # Try to get steps from metadata if available
+                if hasattr(run, 'metadata') and hasattr(run.metadata, 'steps') and run.metadata.steps:
```

Ruff (0.11.9)

Line too long (102 > 100)

(E501)